### ISSUE TITLE
PLEASE READ THIS BEFORE STARTING TO WORK

### ISSUE THREAD

**Issue by hubcio** [iggy-rs/iggy#685]
Multiple people often start working on same problem, which causes synchronization issues. In order to avoid this, please inform us via discord that you want to start work on something. This helps us keep track of who is working on what and minimizes duplication of effort.i

Thank you!

**Comment by 0xg0nz0** [comment]
@hubcio have you thought about using GitHub Projects more? I started playing with it the other day on a personal project and it might help with workflow.

**Comment by hubcio** [comment]
@0xg0nz0 yes, we don't feel need to do it yet, but once we get 10+ regular devs, then we'll probably switch to it
